inception-workload-dynamic/src/main/java/de/tudarmstadt/ukp/inception/workload/dynamic/page/settings/WorkflowAndMonitoringPanel.java

```diff
@@ -43,24 +46,28 @@
     private @SpringBean WorkloadProperties workloadProperties;
     private @SpringBean WorkflowProperties workflowProperties;
 
+    private final String ID;
+
+    private static final long serialVersionUID = -6220828178550562376L;
 
-    public WorkflowAndMonitoringPanel(String aID, IModel<Project> aProject)
-    {
+    public WorkflowAndMonitoringPanel(String aID, IModel<Project> aProject) {
         super(aID, aProject);
 
+        ID = aID;
```

Comment:
Why save the ID when you pass it to the superclass? You should be able to obtain the ID again from the superclass.